system_baseline/validators.py

```diff
@@ -46,6 +46,31 @@ def check_for_empty_name_values(facts):
             raise FactValidationError("value for %s cannot be empty" % fact["name"])
 
 
+def check_for_invalid_whitespace_name_values(facts):
+    """
+    check if any name or values have invalid whitespace at beginning and end; raises an exception.
+    """
+    for fact in facts:
+        if "values" in fact:
+            check_for_invalid_whitespace_name_values(fact["values"])
+        # if "name" in fact and re.search(r'^\s|\s$', fact["name"]):
```

Comment:
looks like there is some unused code here and also below

Reply:
whoops